src/main/MQ2ParseAPI.cpp

```diff
@@ -1,58 +0,0 @@
-/*
- * MacroQuest: The extension platform for EverQuest
- * Copyright (C) 2002-2023 MacroQuest Authors
- *
- * This program is free software; you can redistribute it and/or modify
- * it under the terms of the GNU General Public License, version 2, as published by
- * the Free Software Foundation.
- *
- * This program is distributed in the hope that it will be useful,
- * but WITHOUT ANY WARRANTY; without even the implied warranty of
- * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
- * GNU General Public License for more details.
- */
-
-#include "pch.h"
-#include "MQ2Main.h"
-
-using namespace mq::datatypes;
-namespace mq {
-
-// Forward decls
-namespace datatypes {
-	void InitializeMQ2DataTypes();
-	void ShutdownMQ2DataTypes();
-}
-// From MQ2DataAPI.cpp
-void InitializeMQ2Data();
-void ShutdownMQ2Data();
-
-void InitializeParser()
-{
-	DebugSpewNoFile("InitializeParser()");
-
-	InitializeMQ2DataTypes();
-	InitializeMQ2Data();
-
-	bmParseMacroParameter = AddMQ2Benchmark("ParseMacroParameter");
-}
-
-void ShutdownParser()
-{
-	DebugSpewNoFile("ShutdownParser()");
-
-	ShutdownMQ2Data();
-	ShutdownMQ2DataTypes();
-}
-
-char* ParseMacroParameter(PSPAWNINFO pChar, char* szOriginal, size_t BufferSize)
-{
-	EnterMQ2Benchmark(bmParseMacroParameter);
```

Comment:
I don't personally care about this, but all of your changes derived from the ParseMacroParameter to ParseMacroData lose the EnterBenchMark code.  Just want to be sure that's intentional (I assume it is).

Reply:
no, it shouldn't have been. it should've been the opposite (ParseMacroData didn't use the benchmark.) Will look it over again.